OpenRA.Mods.Common/Widgets/Logic/GameSaveBrowserLogic.cs

```diff
@@ -112,6 +112,7 @@ public GameSaveBrowserLogic(Widget widget, ModData modData, Action onExit, Actio
 				}
 
 				var saveButton = panel.Get<ButtonWidget>("SAVE_BUTTON");
+				saveButton.IsDisabled = () => string.IsNullOrEmpty(saveTextField.Text);
```

Comment:
I suggest
```suggestion
				saveButton.IsDisabled = () => string.IsNullOrWhiteSpace(saveTextField.Text);
```
since we don't want to allow spaces as file names either.